common/lib/common-definitions/src/logger.ts

```diff
@@ -120,3 +120,23 @@ export interface ITelemetryLogger extends ITelemetryBaseLogger {
      */
     shipAssert(condition: boolean, event?: ITelemetryErrorEvent): void;
 }
+
+// ////////////////////// fluidError.ts in common-definitions /////////////////////////////
+/**
+ * Can be decl merged to add stuff like packagename, odspErrorResponse, etc
+ * The logger impl can choose to log anything in here as appropriate, but by default it's not
+ */
+export interface ISensitiveDebugData {
```

Comment:
i think this should also extend ITelemetryProperties

Reply:
It's intentional to be explicitly typed as opposed to using an index signature. Each member that's added to that interface via declaration merging would carry some specific semantics that the base logger implementer may choose to interpret a certain way to decide to log it.